setup.cfg

```
@@ -22,3 +22,7 @@ classifier =
 [files]
 packages =
     gcl_looper
+
+[entry_points]
+console_scripts =
+    looper-launchpad-service = gcl_looper.cmd.launchpad:main
```

```suggestion
    gcl-looper-launchpad-service = gcl_looper.cmd.launchpad:main
```

I think it would be more straightforward

---

Fixed, thanks!